cmake/O2RootMacroExclusionList.cmake

```diff
@@ -53,7 +53,9 @@ list(APPEND O2_ROOT_MACRO_EXCLUSION_LIST
             macro/load_all_libs.C
             macro/putCondition.C
             macro/rootlogon.C
-            Detectors/DCS/test/processor_dpcom_o2.C)
+            Detectors/DCS/test/processor_dpcom_o2.C
+            Detectors/FIT/FT0/calibration/macros/makeDummyFT0CalibObjectInCCDB.C)
```

Comment:
what is the reason to exclude the macro from the test?

Reply:
I think it will be deleted anyway in next versions of this workflow, so it is temporary.
BTW I didn't think much about this file, CMake was complaining about tests so I added this file to not tested macros list.
I can also remove this macro from mentioned list and make a test for it.